src/main/java/org/folio/circulation/domain/notice/TemplateContextUtil.java

```diff
@@ -217,6 +217,9 @@ private static JsonObject createRequestContext(Request request) {
     optionalRequest
       .map(Request::getAddressType)
       .ifPresent(value -> requestContext.put("deliveryAddressType", value.getName()));
+    optionalRequest
```

Comment:
I think I might be tempted to put a blank line between this and the previous statement

Reply:
Sure, done.